activerecord/lib/active_record/conversion.rb

```diff
@@ -0,0 +1,16 @@
+module ActiveRecord
+  module Conversion # :nodoc:
+    include ActiveModel::Conversion
+
+    # Wrapper around <tt>ActiveModel::Conversion#to_model</tt> to return an
+    # instance of the <tt>ActiveRecord::Errors</tt> class for the model's
+    # errors.
+    def to_model
+      @errors = Errors.new(self)
+      # fill validation errors
+      validate
```

Comment:
Do we need to call validations here? Should not it only be called in the same place of the normal flow?